drift/core/telescope.py

```diff
@@ -171,6 +179,9 @@ class TransitTelescope(with_metaclass(abc.ABCMeta, config.Reader, ctime.Observer
     freq_lower = config.Property(proptype=float, default=400.0)
     freq_upper = config.Property(proptype=float, default=800.0)
     num_freq = config.Property(proptype=int, default=50)
+    use_chime_freq = config.Property(proptype=bool, default=True)
```

Comment:
This doesn't need to be quite so specific. You can mostly do this with the parameters you have in here by adding a `freq_mode` parameter like exists in `cora-makesky`